third_party/lua/luadebug.lua

```diff
@@ -788,6 +788,19 @@ function dbg.call(f, ...)
     end, ...)
 end
 
+-- interactive console start
+function dbg.start(path, ...)
+    -- all checks should be done in init.c
+    assert(path)
```

Comment:
Maybe it makes sense to use `assert(path ~= nil and type(path) == 'string'`?

Reply:
Good point

Reply:
Done, see above.